lxc/export.go

```diff
@@ -107,8 +108,18 @@ func (c *cmdExport) Run(cmd *cobra.Command, args []string) error {
 	}
 
 	// Get name of backup
-	backupName := strings.TrimPrefix(op.Get().Resources["backups"][0],
-		"/1.0/backups/")
+	uStr := op.Get().Resources["backups"][0]
+	u, err := url.Parse(uStr)
+	if err != nil {
+		fmt.Println("Invalid URL", uStr)
+		return err
+	}
+
+	backupName, err := url.PathUnescape(path.Base(u.EscapedPath()))
+	if err != nil {
+		fmt.Println("Invalid backup name segment in path", u.EscapedPath())
```

Comment:
Lets not print to stdout when we should be returning an error (that will go to stderr).

`return fmt.Errorf(`Invalid backup name segment in path %q`, u.EscapedPath())`